### DISCUSSION TITLE
Application delete

### DISCUSSION THREAD

**Discussion by izzie88** [argoproj/argo-cd#7592] [Q&A]
Hello,

Is it possible to delete an application on Argocd but the services wont be deleted on my k8s cluster.

The first time i did  deleted an application on argocd, the application and all my services was deleted from the cluster.

Any way around this ?

Please help

**Comment by pasha-codefresh** [comment]
Hello @izzie88 , Afaik , you should use the non-cascading option according to doc 
https://argo-cd-docs.readthedocs.io/en/latest/user-guide/app_deletion/
